app/parse.py

```diff
@@ -15,9 +39,61 @@ class Product:
     num_of_reviews: int
 
 
+def parse_single_product(product: WebElement):
+    return Product(
+        title=product.find_element(By.CLASS_NAME, "title").text,
+        description=product.find_element(By.CLASS_NAME, "description").text,
+        price=float(product.find_element(By.CLASS_NAME, "price").text.replace("$", "")),
+        rating=len(product.find_elements(By.CLASS_NAME, "ws-icon-star")),
+        num_of_reviews=int(
+            product.find_element(By.CLASS_NAME, "review-count").text.split()[0]
+        ),
+    )
+
+
+def parse_page_for_products(page_name: str, url: str):
+    chrome_options = Options()
+    chrome_options.add_argument("--headless")
+    driver = webdriver.Chrome(options=chrome_options)
```

Comment:
Ensure that the WebDriver is properly closed after use. Consider using a try-finally block or a context manager to guarantee that driver.quit() is called, even if an exception occurs.